### PR TITLE
Add resdk.tables.MATables for microarray data support

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: src/resdk/tables/microarray.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+        # Fetch collection microarray and metadata
+        tables = MATables(collection)
+        meta = tables.meta
+        exp = tables.exp
+
+    """
+
+    process_type = "data:microarray:normalized"
+    EXP = "exp"
```
Comment (robertcv) [inline src/resdk/tables/microarray.py:48]: I would suggest using some other label like `"ma"`. Although the cached file name will not be the same as the hash of data objects is different I would still like it to be different so it's more obvious which cache file contains what.

File: src/resdk/tables/microarray.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
+""".. Ignore pydocstyle D400.
+
+========
```
Comment (robertcv) [inline src/resdk/tables/microarray.py:3]: You should also add some lines to the topic documentation.

**Review by robertcv — CHANGES_REQUESTED** [review]

File: src/resdk/tables/microarray.py (hunk 2 above)
Comment (JureZmrzlikar) [reply src/resdk/tables/microarray.py:3]: Good point, thanks!

File: src/resdk/tables/microarray.py (hunk 1 above)
Comment (JureZmrzlikar) [reply src/resdk/tables/microarray.py]: Done.

**Comment by JureZmrzlikar** [comment]
@robertcv I've addressed your comments, please re-review.

**Review by robertcv — CHANGES_REQUESTED** [review]
The changes look OK. I just noticed one more thing. I would like it for exp to include the `attrs` attribute similar to RNATables. I would add parameters `exp_type` and `platform` from the data object output attributes.

**Review by robertcv — APPROVED** [review]

**Review by gregorjerse — APPROVED** [review]

**Pull request merged**